From c0d171dd2c21aa1713d4ca61ac590615d1328d8b Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 22 Mar 2023 09:53:06 +0100
Subject: [PATCH 0251/1695] dt-bindings: add RISAF to default bindings config
 for stm32mp25

The RIFSC header is now part of default bindings header file for
stm32mp25.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 239058c50134844aad30ac279aa7a0af1d07cbcf dt-bindings: add RISAF to default bindings config for stm32mp25]
Change-Id: I4b5a62cda28275bc3b26e1c0ac4f2fc7c8a530fc
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
---
 core/include/drivers/stm32mp_dt_bindings.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/include/drivers/stm32mp_dt_bindings.h b/core/include/drivers/stm32mp_dt_bindings.h
index 710fa2646..bbff1d8ff 100644
--- a/core/include/drivers/stm32mp_dt_bindings.h
+++ b/core/include/drivers/stm32mp_dt_bindings.h
@@ -29,6 +29,7 @@
 #include <dt-bindings/reset/stm32mp25-resets.h>
 #include <dt-bindings/soc/stm32mp25-rif.h>
 #include <dt-bindings/soc/stm32mp25-rifsc.h>
+#include <dt-bindings/soc/stm32mp25-risaf.h>
 #endif
 
 #endif /* __DRIVERS_STM32MP_DT_BINDINGS_H */
-- 
2.47.2

